changes.tmp

```diff
@@ -15,6 +15,13 @@ Added	The functions "to_utf8()" and "to_ansi()" can be used to convert to and fr
 Added	Folding has been enabled for LaTeX files.
 Added	The terminal now also proposed global variables for auto completion.
 Applied	The revision of large files won't be created by calculating a DIFF. Instead, the whole file content is saved.
+Added	New table methods for insertion of cells ("TAB().insertXYZ()"), removing of cells ("TAB().removeXYZ()") and removing unnecessary trailing cells and columns ("TAB().shrink") have been added.
```

Comment:
Not really needed to be fixed, but a question. 

Is it okay to add the revision in the middle of the changes.tmp? cause it's no longer chronological

Reply:
That's totally fine. In the end, the entries will be sorted alphabetically nevertheless.